File: src/main/java/com/rs/java/game/area/multi/KalphiteLairMulti.java
package com.rs.java.game.area.multi;

import com.rs.java.game.area.Area;

public final class KalphiteLairMulti extends Area {

	@Override
	public Area update() {
		return this;
	}

	@Override
	public String name() {
		return "Kalphite Lair Multi";
	}

	public KalphiteLairMulti() {
		super(13972);
	}

	@Override
	public boolean member() {
		return false;
	}

	@Override
	public Environment environment() {
		return Environment.MULTI;
	}

}

File: src/main/java/com/rs/kotlin/game/world/area/areas/multi/KalphiteLairMulti.kt
package com.rs.kotlin.game.world.area.areas.multi

import com.rs.kotlin.game.world.area.Area

class KalphiteLairMulti : Area(13972) {
    override fun update(): Area {
        return this
    }

    override fun name(): String {
        return "Kalphite Lair Multi"
    }

    override fun member(): Boolean {
        return false
    }

    override fun environment(): Environment {
        return Environment.MULTI
    }
}
